xcube_clms/utils.py

```diff
@@ -0,0 +1,262 @@
+# The MIT License (MIT)
```

Comment:
It seems that the functions in this file help implementing very specific tasks. If such functions are not used twice please define them in the module where they are used. `get_authorization_header` and `spinner` do not have much in common and belong to different contexts.

Reply:
b637c7714